private/buf/cmd/buf/command/convert/no_resolve_message_set.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2020-2024 Buf Technologies, Inc.
+//
```

Comment:
this type of logic should not live in `private/buf/cmd/buf` - as much as possible, very little logic should live here. this needs to find the appropriate package to live in, its a component of the library